lib/select-builder.js

```diff
@@ -46,35 +22,15 @@ const checkType = (value, name, type) => {
   }
 };
 
-const makeParamValue = (cond, value, params) => {
-  if (value instanceof QueryBuilder) {
-    return '(' + value.build(params) + ')';
-  } else if (cond === 'IN' || cond === 'NOT IN') {
-    const conditions = mapJoinIterable(value, v => params.add(v), ', ');
-    return '(' + conditions + ')';
-  } else if (cond && cond.endsWith('ANY')) {
-    return '(' + params.add(value) + ')';
-  } else if (cond && cond.includes('BETWEEN')) {
-    const [start, end] = value;
-    const startParam =
-      start instanceof QueryBuilder ? `(${start.build()})` : params.add(start);
-    const endParam =
-      end instanceof QueryBuilder ? `(${end.build()})` : params.add(end);
-    return `${startParam} AND ${endParam}`;
-  } else {
-    return params.add(value);
-  }
-};
-
 class SelectBuilder extends QueryBuilder {
   constructor(params, options) {
     super(params, options);
+    this.conditionsBuilder = new ConditionsBuilder(params, options);
```

Comment:
It's not clear from the name which conditions this builder is for, I'd propose to rename it:
```suggestion
    this.whereConditions = new ConditionsBuilder(params, options);
```